scripts/lint/check-user-facing-strings.sh

```
@@ -0,0 +1,37 @@
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [[ "${TRACE-0}" == "1" ]]; then set -o xtrace; fi
+
+echo "==> checking for user-facing string literals..."
+
+# Find user-facing strings in src/ (exclude: tests, debug, comments, log messages)
+# Pattern: strings in cout, printf, error messages, UI prompts
+count=$(find src -name '*.cpp' -o -name '*.h' | \
```

Shellcheck (0.11.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)